packs/boundary/metadata.hcl

```diff
@@ -0,0 +1,11 @@
+app {
+  url    = "https://www.boundaryproject.io/"
+  author = "paladin-devops"
```

Comment:
I'll change post-merge, but this should actually be the app author, in this case HashiCorp :)